foundations/src/telemetry/settings/logging.rs

```diff
@@ -66,21 +65,58 @@ pub enum LogFormat {
     Json,
 }
 
-/// Verbosity level of the log.
+/// Log verbosity levels which match 1:1 with [`slog::Level`].
 #[derive(Clone, Debug, Copy)]
-pub struct LogVerbosity(pub Level);
+pub enum LogVerbosity {
+    /// See [`slog::Level::Critical`].
+    Critical,
+    /// See [`slog::Level::Error`].
+    Error,
+    /// See [`slog::Level::Warning`].
+    Warning,
+    /// See [`slog::Level::Info`].
+    Info,
+    /// See [`slog::Level::Debug`].
+    Debug,
+    /// See [`slog::Level::Trace`].
+    Trace,
+}
 
 impl Default for LogVerbosity {
     fn default() -> Self {
-        Self(Level::Info)
+        Self::Info
```

Comment:
nit: you can use derive `Default` in combination with `#[default]` attribute for the variant instead